### PR TITLE
fix coding style for test files

### PR DESCRIPTION
**Pull request by mimmi20**

### CODE REVIEW

File: test/classes/example/tasks/TestEcho.php [hunk 1]
```diff
@@ -0,0 +1,30 @@
+<?php
+
+/*
```
Comment (siad007) [inline test/classes/example/tasks/TestEcho.php]: Would you mind changing the file comments to `/**` and omit the empty line 2.

Comment (mimmi20) [reply test/classes/example/tasks/TestEcho.php]: fixed

Comment (siad007) [reply test/classes/example/tasks/TestEcho.php]: > 
> 
> fixed

no

Comment (mimmi20) [reply test/classes/example/tasks/TestEcho.php]: I found more files with this type of doc block and fixed them all. I added this doc block to files where it was missing.

**Comment by siad007** [comment]
@mimmi20 thx 👍

**Pull request merged**